app/scenes/HomeScreen/selector.js

```diff
@@ -2,7 +2,8 @@ import { createSelector } from 'reselect';
 import get from 'lodash/get';
 import { initialState } from './reducer';
 
-export const selectHomeDomain = state => (state.example || initialState).toJS();
+export const selectHomeDomain = state =>
+  (state.selectTracks || initialState).toJS();
```

Comment:
lets discuss about this , then we will update the PR,  as this won't work `state.selectTracks`